### ISSUE TITLE
Make svcat version command more like kubectl version

### ISSUE THREAD

**Issue by jboyd01** [kubernetes-incubator/service-catalog#2092]
Make `svcat version` more in line with `kubectl version` by always reporting the client version.  

Presently svcat reports the following for version info when Kubernetes isn't running:
```console
$ svcat version
Error: unable to get version, Get http://localhost:8080/version: dial tcp [::1]:8080: getsockopt: connection refused
$ svcat version --client
client: v0.1.17-7-gef904a5-dirty
```

vs kubectl

```console
$ kubectl version
Client Version: version.Info{Major:"1", Minor:"11+", GitVersion:"v1.11.0-alpha.2.8+0d43bdec2b8598-dirty", GitCommit:"0d43bdec2b8598ff542a1afdee876d417b4e7668", GitTreeState:"dirty", BuildDate:"2018-05-16T20:04:10Z", GoVersion:"go1.10.1", Compiler:"gc", Platform:"linux/amd64"}
The connection to the server localhost:8080 was refused - did you specify the right host or port?
```

**Comment by carolynvs** [comment]
To be clear, `svcat version` _always_ returns the client version, and the `--client` flag allows us to query the client version only, in instances where perhaps there isn't a configured cluster yet.

The problem is that when we fail to query the server version, svcat exits immediately before the client version is also printed. 

The code that needs to be changed is in the version command

https://github.com/kubernetes-incubator/service-catalog/blob/a514e67a664cff404196158436e6d158a33fc886/cmd/svcat/versions/version_cmd.go#L68-L83

It could be fixed in either of two ways:
1. Print the error message and do not exit immediately when `c.App.ServerVersion()` returns an error, so that `output.WriteVersion(c.Output, client, server)` is still called. The function should still return an error when that call fails however, so that the return code of the `svcat version` function is still non-zero.
2. Change the function so that we print out the versions incrementally, first the client version, then the server version.

**Comment by carolynvs** [comment]
I thought that this would be a good first PR initially, but since the tests don't exist yet this may be easier for someone already familiar with writing tests in service catalog, or a very enterprising new contributor! 😀

**Comment by A-Hilaly** [comment]
Hello :) ! Can i work on this this ?

**Comment by carolynvs** [comment]
@A-Hilaly Yup! Go for it 🙌 When you get to the part about adding tests, feel free to ping back for help. As I noted originally, this part doesn't have existing tests so you will need to add them yourself.

**Comment by A-Hilaly** [comment]
Alright, thank you 🙏 I'll open a PR soon !

**Comment by jboyd01** [comment]
closed by #2109
